source/release-notes/4-1-4.rst

```diff
@@ -0,0 +1,29 @@
+.. index:: CrafterCMS 4.1.4 Release Notes
+
+----------------
+CrafterCMS 4.1.4
+----------------
+
+Released *TBD*
+
+This release includes fixes for issues with the release version 4.1.3
+
+.. important::
+
+    Please see :ref:`Breaking changes <breaking-changes-4-1-x>` for breaking changes and other
+    things to consider before upgrading
+
+    For instructions on how to update your CrafterCMS install to 4.1.0 (applies to 4.1.4),
+    please see :ref:`upgrading-craftercms`
+
+^^^^^^^^^
+Bug Fixes
```

Comment:
Let's also include this https://github.com/craftersoftware/craftercms/issues/829

Reply:
I still don't see this.